packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -24,8 +24,8 @@ dependencies:
     sdk: flutter
   image_picker_android: ^0.8.4+11
   image_picker_for_web: ^2.1.0
-  image_picker_ios: ^0.8.4+11
-  image_picker_platform_interface: ^2.3.0
+  image_picker_ios: ^0.8.6
```

Comment:
This should be `0.8.6+1` now to pick up the last fix.

Reply:
I'm still waiting for this version to be published on pub.dev. It seems that the release action failed.

Reply:
Oops, sorry about that; I forgot to check that publishing had completed. It looks like there was a flake in the analysis run during post-submit tests, and it's already being re-run.

Reply:
Now it's live. Sorry for the delay!

Reply:
Should be good to go now, thanks!